aspnetcore/grpc/authn-and-authz/sample/6.x/TicketerClient/TicketerClient.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp6.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Google.Protobuf" Version="3.20.0" />
+    <PackageReference Include="Grpc.Net.Client" Version="2.44.0" />
+    <PackageReference Include="Grpc.Tools" Version="2.45.0" PrivateAssets="All" />
```

Comment:
```suggestion
    <PackageReference Include="Grpc.Net.Client" Version="2.43.0" />
    <PackageReference Include="Grpc.Tools" Version="2.43.0" PrivateAssets="All" />
```

Reply:
Standardize version on 2.43.0 (2.44.0 has a known issue and 2.45.0 isn't out yet for all packages)